pkg/color/formatter.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package color
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var (
+	// ColoredOutput will check if color escape codes should be printed. This can be changed
+	// for testing to an arbitrary method.
+	ColoredOutput = coloredOutput
+	// IsTerminal will check if the specified output stream is a terminal. This can be changed
+	// for testing to an arbitrary method.
+	IsTerminal = isTerminal
+	// NoColor allow to force to not output colors.
+	NoColor = false
+)
+
+// Color can be used to format text using ANSI escape codes so it can be printed to
+// the terminal in color.
+type Color int
+
+var (
+	// LightRed can format text to be displayed to the terminal in light red, using ANSI escape codes.
```

Comment:
I think you can remove these comments. it's pretty clear what each of these colors is doing, and is mostly visual clutter :) maybe one comment above the start of the `var` section about how these are all ANSI escape codes and that's it.